code/modules/clothing/shoes/marine_shoes.dm

```diff
@@ -21,6 +21,10 @@
 	var/armor_stage = 0
 	items_allowed = list(/obj/item/attachable/bayonet, /obj/item/weapon/melee/throwing_knife, /obj/item/weapon/gun/pistol/holdout, /obj/item/weapon/gun/pistol/m43pistol, /obj/item/tool/screwdriver)
 
+/obj/item/clothing/shoes/marine/Initialize(mapload, ...)
```

Comment:
should also apply to shoes/veteran (PMC shoes) as well as /obj/item/clothing/shoes/combat and /obj/item/clothing/shoes/souto as they are armoured